packages/react-native-renderer/src/ReactFabricRenderer.js

```diff
@@ -203,11 +221,28 @@ const ReactFabricRenderer = ReactFiberReconciler({
     return false;
   },
 
-  getRootHostContext(): {} {
-    return emptyObject;
+  getRootHostContext(rootContainerInstance: Container): HostContext {
+    return __DEV__ ? {isInAParentText: false} : emptyObject;
   },
 
-  getChildHostContext(): {} {
+  getChildHostContext(
+    parentHostContext: HostContext,
+    type: string,
+  ): HostContext {
+    if (__DEV__) {
+      const oldIsInAParentText = parentHostContext.isInAParentText;
```

Comment:
Why this way vs what you had before?

Reply:
To avoid creating new wrapper objects if `isInAParentText` hasn't changed.